src/components/20-molecules/top-content-bar/index.scss

```diff
@@ -1,5 +1,95 @@
+$mediaquery-mobile-up: 411px;
```

Comment:
@markus-walther 
In the design https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=8744%3A5711 we have breakpoint: mobile < 411px and tablet < 991px
I agree that I can use existing $breakpoint-md:992px;
but for mobile I had to set the new breakpoint.
Should I use existing $breakpoint-xs:576px; instead of 411px ?

Reply:
I can't say what motivated the designers to specify an even smaller mobile breakpoint - can you ask if that was deliberate or an oversight?

If oversight, use $breakpoint-xs, otherwise your new breakpoint (which you'd keep local to this component).

Please record the designer's decision in the PR description, so we know.